components/Bubbles/BubblesGame.js

```diff
@@ -316,12 +334,28 @@ class BubblesGame extends React.Component {
                 animationFrameIndex={this.state.bubbleAnimationIndex}
                 loopAnimation={false}
                 coordinates={{top: 400 * this.scale.screenHeight,
-                  left: 40 * this.scale.screenWidth}}
+                  left: -300 * this.scale.screenWidth}}
                 size={{ width: Math.floor(300 * this.scale.image),
                   height: Math.floor(285 * this.scale.image)}}
               />
             : null}
 
+            <AnimatedSprite
+              ref={'monsterRef'}
+              character={monster}
+              characterUID={this.characterUIDs.monster}
+              animationFrameIndex={this.state.monsterAnimationIndex}
+              tweenStart={'fromCode'}
+              tweenOptions={this.monster.tweenOptions}
+              onTweenFinish={(characterUID)=> this.onCharacterTweenFinish(characterUID)}
+              loopAnimation={this.monster.loopAnimation}
+              coordinates={{top: (510) * this.scale.screenHeight,
+                left: -300 * this.scale.screenWidth}}
+              size={{ width: Math.floor(300 * this.scale.image),
```

Comment:
instead of Math.floor(214 * this.scale.image) use Math.floor(monster.size.width * this.scale.image). Technically I think we can ignore Math.floor and it gets done automatically.